packages/eas-cli/package.json

```diff
@@ -30,8 +31,10 @@
     "chalk": "4.1.2",
     "cli-table3": "0.6.1",
     "dateformat": "4.6.3",
+    "dicer": "0.3.1",
```

Comment:
Sizes look OK to me:

- dicer: 2.4KB min+gz (incl. deps)
- fast-deep-equal: 414B min+gz
- structured-headers: 2.7KB min+gz